src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: $card-width;
+  background-color: $color-white;
+  border: 1px solid $color-border;
+  border-radius: $border-radius;
+  overflow: hidden;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  font-family: $font-primary;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    object-fit: contain;
+    margin-top: 32px;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    color: $color-text;
+    text-align: left;
+    margin: 0 16px 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $color-text2;
+    margin-bottom: 16px;
+    margin-left: 34px;
+    width: 100%;
+    text-align: left;
+  }
+
+  &__rating {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    padding: 0 16px;
+    margin-bottom: 24px;
+
+    .stars {
+      margin-left: 16px;
+    }
+
+    .card__reviews {
+      font-size: 10px;
+      color: $color-text;
+      text-align: right;
+      padding-right: 16px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    margin-bottom: 16px;
+
+    .card__price-label {
+      font-size: 12px;
+      color: $color-text2;
+      padding: 0 16px;
+    }
+
+    .card__price-value {
+      font-size: 17px;
+      font-weight: 700;
+      color: $color-text;
+      padding-right: 16px;
+    }
+  }
+
+  &__button {
+    width: calc(100% - 54px);
+    padding: 12px;
+    background-color: $color-botton;
```

Comment:
There is a typo in the variable name `$color-botton`. It should be corrected to `$color-button` to ensure the correct color is applied to the button background.